tests/Fixer/PhpUnit/PhpUnitTestClassRequiresCoversFixerTest.php

```diff
@@ -448,4 +448,62 @@ public static function provideFix82Cases(): iterable
                 PHP,
         ];
     }
+
+    /**
+     * @dataProvider provideFixUnit11Cases
+     *
+     * @requires PHP 8.3
```

Comment:
Why PHP 8.3 is required? Test cases does not use anything from this specific version, there are regular attributes and regular PHP 8.0 code. We use this only if tested code includes syntax introduced in specific PHP version (like `testFix82` tests `readonly` classes, introduced in 8.2). This new cases should just go to `provideFix80Cases` and cover additional attribute(s).

Also, I am not sure if this should go under `FixUnit11` (which enforced a change in `tests/Test/AbstractFixerTestCase.php`). This is not a convention used in this test class and in the project.

Reply:
Because PHPUnit 11 has a specific requirement for PHP 8.3, that's why I choose to do it this way. I must say, I indeed was not sure what the best course of action was. I will implement the test cases within the `provideFix80Cases`

Reply:
As far as I see [PHPUnit 11 requires PHP 8.2](https://packagist.org/packages/phpunit/phpunit#11.1.0), but we don't care about it - we assume that if someone uses `CoversMethod`, then correct PHPUnit is installed, dependencies are resolved, and Fixer is run using required PHP. In our test cases we only care about PHP syntax features, so specific tests are run by CI matrix only for supported PHP versions.

Reply:
Oopsie, just noticed, I was looking at the main branch, indeed the latest 11.* tag is still `>=8.2`